src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+.page {
+  font-family: $main-font-family;
+  color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used here. Ensure this variable is defined in your SCSS files.